frontend/src/views/OtherUses/AddEditOtherUses.jsx

```diff
@@ -107,6 +107,38 @@ export const AddEditOtherUses = () => {
     }
   }
 
+  const onCellValueChanged = useCallback(
+    async (params) => {
+      if (params.colDef.field === 'provisionOfTheAct') {
```

Comment:
If this only applies to a single value, it should be in the valueSetter of that column in _schema